# clang seems to not remove completely unused ref when removing dead code
#
# http://lists.busybox.net/pipermail/busybox/2013-January/078871.html
# http://lists.busybox.net/pipermail/busybox/2013-January/078884.html
#
diff -ru busybox-1.25.0-orig/archival/libarchive/Kbuild.src busybox-1.25.0/archival/libarchive/Kbuild.src
--- busybox-1.25.0-orig/archival/libarchive/Kbuild.src
+++ busybox-1.25.0/archival/libarchive/Kbuild.src
@@ -45,7 +45,7 @@
 
 lib-$(CONFIG_AR)                        += get_header_ar.o unpack_ar_archive.o
 lib-$(CONFIG_CPIO)                      += get_header_cpio.o
-lib-$(CONFIG_TAR)                       += get_header_tar.o unsafe_prefix.o
+lib-$(CONFIG_TAR)                       += get_header_tar.o unsafe_prefix.o decompress_uncompress.o
 lib-$(CONFIG_FEATURE_TAR_TO_COMMAND)    += data_extract_to_command.o
 lib-$(CONFIG_LZOP)                      += lzo1x_1.o lzo1x_1o.o lzo1x_d.o
 lib-$(CONFIG_LZOP_COMPR_HIGH)           += lzo1x_9x.o
